orchestrator.go

```diff
@@ -1 +1,32 @@
+package main
 
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/trilogy-group/cloudfix-linter/logger"
+)
+
+func extractModulePaths(jsonString string) ([]string, error) {
```

Comment:
add unit tests, please. They will help you find bugs much earlier. It may feel like it slows you down in the short term, but from a project completion standpoint they speed you up.